api/envoy/config/common/dynamic_forward_proxy/v2alpha/dns_cache.proto

```diff
@@ -46,7 +46,8 @@ message DnsCacheConfig {
   //
   //  The returned DNS TTL is not currently used to alter the refresh rate. This feature will be
   //  added in a future change.
```

Comment:
Could you add a comment here saying the value is rounded up to the nearest ms / minimum non-zero? 
(maybe while you're at it write the default 60)

Reply:
Will add the rounding comment.
The default (60) is already mentioned at the beginning of the comment:
https://github.com/envoyproxy/envoy/blob/c05e95c1d835dfe756436ef7d43ad343dada01f3/api/envoy/config/common/dynamic_forward_proxy/v2alpha/dns_cache.proto#L43

BTW: as the implementation only supports the milliseconds granularity, it might be better to change this type to a duration that is only milliseconds and seconds (not sure if one already exists).

Reply:
We use Duration across the API with constraints to achieve this, there isn't a standard proto type with lowered precision.